lib/package.json

```diff
@@ -72,13 +77,17 @@
     "watch": "(trap 'kill 0' SIGINT; npm run build && (npm run build:watch & npm run test -- --watch))"
   },
   "dependencies": {
+    "@connectrpc/connect": "^2.0.2",
```

Comment:
These have been added as dependencies and are re-exported in `lib/src/platform.ts`. This way we can ensure that the generated client services will work with the connectrpc web framework.

One alternative I've considered here is to declare them as `peerDependencies` and actually **not** re-export them. When someone consumes `@opentdf/sdk` they will get a warning that `@connectrcp` needs to be added to their `package.json`. 
I would have pushed for this option if the bundle size increased, but it has not. I think we are good with re-export